test/TritonGPU/dot-operands.mlir

```diff
@@ -302,3 +302,69 @@ module attributes {"ttg.target" = "cuda:90", "ttg.num-ctas" = 1 : i32, "ttg.num-
     tt.return %r : tensor<128x64xf32, #mma>
   }
 }
+
+// -----
+
+#blocked = #ttg.blocked<{sizePerThread = [2, 2], threadsPerWarp = [2, 16], warpsPerCTA = [4, 1], order = [1, 0]}>
+#mma = #ttg.nvidia_mma<{versionMajor = 2, versionMinor = 0, warpsPerCTA = [4, 1], instrShape = [16, 8]}>
```

Comment:
Can we modify one of these tests to use MMAV3? We want to make sure that this stands for both and this would provide more coverage.

Reply:
Sure, I added an extra for mmav3

Reply:
Thanks for the additional tests!